appinventor/appengine/src/com/google/appinventor/client/Ode.java

```diff
@@ -2580,6 +2506,14 @@ public static native void CLog(String message) /*-{
     console.log(message);
   }-*/;
 
+  public static native boolean isMobile() /*-{
```

Comment:
It looks like this is used mainly in CSS. It is more appropriate to use CSS media queries to detect the screen size rather than trying to guess the size based on the user agent. If we do decide to continue down this road, please move this to a utility class rather than dumping it into Ode.

Reply:
Now I am not seeing this used anywhere in the code base. Please remove it.

Reply:
> It looks like this is used mainly in CSS. It is more appropriate to use CSS media queries to detect the screen size rather than trying to guess the size based on the user agent. If we do decide to continue down this road, please move this to a utility class rather than dumping it into Ode.

This is another artifact of the prototype. The mobile mode wasn't really far enough to do anything with, so it wasn't brought over.